lib/ansible/modules/messaging/rabbitmq/rabbitmq_policy.py

```diff
@@ -125,7 +125,10 @@ def _exec(self,
     def _rabbit_version(self):
         status = self._exec(['status'], True, False, False)
 
-        version_match = re.search('{rabbit,".*","(?P<version>.*)"}', status)
+        version_match = re.search(
+            r'({rabbit,".*","|RabbitMQ version:)\s?(?P<version>[0-9.]{3,})("})?',
```

Comment:
In my point of view, it is better to use `rabbitmqctl version` command. It was introduced in some 3.7 release. It seems `status` output is an unreliable source of information - status format can be changed from one release to another.

I suggest the following workflow:

* try to find a version in `rabbitmqctl version` (use --node option like with other commands)
* if this command return error - try `status` with old regex

Anyway, I'm not maintainer here ) I don't think that my approval can change anything.